src/converters/commonMetadata.ts

```diff
@@ -10,6 +10,16 @@ const commonMetadata = (shape : AnySchema, jsonSchema : JSONSchema7) : JSONSchem
     jsonSchema.examples = [shape.spec.meta.example];
   if (shape.spec.meta?.examples)
     jsonSchema.examples = shape.spec.meta.examples;
+  
+  try {
+    if (jsonSchema.type && jsonSchema.type !== 'object') {
```

Comment:
I think default value can be applied to all types , why should we restrict only types other than object ?